### DISCUSSION TITLE
Sending scripts via serial like MTPutty

### DISCUSSION THREAD

**Discussion by stevoid17** [Eugeny/tabby#4900] [Q&A]
Hey all

I was wondering if there's a way of sending a script/config to devices via serial line by line, like the 'send script' option in MTPutty that stops serial adapters freaking out at the amount of data?

If not, it should definitely be added to the list!

Thanks

Steve

**Comment by Eugeny** [comment]
There's no way to do this right now, but I've opened a ticket :+1:

**Reply by stevoid17** [reply]
That's brilliant!

Happy to donate a bit of coin for this sort of thing. That's all this program is missing! Every other serial program sucks compared to this!

**Reply by williamblair333** [reply]
What are you thoughts on doing a bounty / kickstarter / gofundme something something to implement this?

**Reply by Eugeny** [reply]
Hey, I think it's actually been implemented a year or two ago :) Look for "slow send" in your serial profile settings
